src/Utils/request/errorHandler.ts

```diff
@@ -76,6 +82,24 @@ type PydanticError = {
   url: string;
 };
 
+function isStructuredError(cause: HTTPErrorCause): boolean {
+  return (
+    cause !== undefined && typeof cause === "object" && !Array.isArray(cause)
+  );
+}
+
+function handleStructuredErrors(cause: HTTPErrorCause) {
+  if (cause && typeof cause === "object" && !Array.isArray(cause)) {
+    Object.values(cause).forEach((value) => {
+      if (Array.isArray(value)) {
+        value.forEach((err) => {
+          toast.error(`${err}`);
+        });
+      }
```

Comment:
and what if it's not an array, but a string? that also needs to be handled right?